File: youtube learning/c programming 1/CHAPTER_05_PRACTICE_SET/05_problem.c
#include <stdio.h>

int sum_of_natural_number(int);

int sum_of_natural_number(int n)
{
  if (n == 1)
  {
    return 1;
  }
  return sum_of_natural_number(n - 1) + n;
}

int main()
{
  printf("The sum of frist natural number is %d", sum_of_natural_number(6));
  return 0;
}

File: youtube learning/Python/Chapter 7 - PS/05_problem.py
# 5. Write a program to find the sum of first n natural numbers using while loop.

n = int(input("Enter the number: "))
i = 1
sum = 0

while i <= n:
  sum += i
  i += 1

print(f"sum of first {n} natural numbers is ",sum)

# OUTPUT:
"""
Enter the number: 10
sum of first 10 natural numbers is  55

"""